collect_app/src/main/java/org/odk/collect/android/utilities/EncryptionUtils.java

```diff
@@ -355,8 +356,7 @@ public static EncryptedFormInformation getEncryptedFormInformation(Uri uri,
         // submission must have an OpenRosa metadata block with a non-null
         // instanceID value.
         if (instanceMetadata.instanceId == null) {
-            Timber.e("No OpenRosa metadata block or no instanceId defined in that block");
-            return null;
+            throw new EncryptionException("This form does not specify an instanceID. You must specify one to enable encryption.", null);
```

Comment:
I think we should probably have that string in `string.xml`

Reply:
Putting strings in `string.xml` is not "free" in the sense that it adds more work for translators. I think in the case of edge cases like this one, it's preferable to avoid having it translated. This is extremely unlikely since pyxform and Build add instanceID. It's really only likely when someone comes across a really old form definition (generally used for testing).

I've added 557a826 to explain the exception and hopefully you find it reasonable.